codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/generators/PythonServerStructureGenerator.kt

```diff
@@ -40,7 +40,7 @@ class PythonServerStructureGenerator(
     private val codegenContext: ServerCodegenContext,
     private val writer: RustWriter,
     private val shape: StructureShape,
-) : StructureGenerator(model, codegenContext.symbolProvider, writer, shape, emptyList()) {
+) : StructureGenerator(model, codegenContext.symbolProvider, writer, shape, emptyList(), codegenContext.structSettings()) {
```

Comment:
If there is no chance for the user to set this flag on the server side then this is okay. Otherwise, I think let's set this to:  `StructSettings(flattenVecAccessors = false)` for now.

Reply:
a customer can set this on the server side with the smithy-build.json setting

Reply:
Keep it like it is. I wrongly thought that the underlying field is also going to get changed. Only the accessor gets changed, so this is good.